### PR TITLE
Hide Accordion closed Sections instead of unmounting them

### PR DESCRIPTION
**Pull request by supernova-at**
## Description

This PR hides accordion section contents that aren't open instead of not rendering them at all.
This saves components from mounting / unmounting on section open changes, but may overfetch (if a section is never opened, its contents may still make network calls).

This PR also sets all price adjustment accordion sections to closed initially, which is the intended UX.

## Related Issue

Closes PWA-360.

### Verification Steps

1. Go to the `/cart` page
1. Inspect element on the price adjustments `Accordion`
1. See that the contents of the sections that are closed still appear in the DOM, they're just hidden from the user

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.

### CODE REVIEW

**Comment by sirugh** [comment]
<3 Perfect. Fix the tests and I'll approve :D

**Comment by supernova-at** [comment]
PR Updated:
  * Fixes merge conflicts

File: packages/venia-ui/lib/components/CartPage/PriceAdjustments/__tests__/priceAdjustments.spec.js [hunk 1]
```diff
@@ -3,7 +3,9 @@ import { createTestInstance } from '@magento/peregrine';
 
 import PriceAdjustments from '../priceAdjustments';
 
+jest.mock('../CouponCode', () => 'CouponCode');
 jest.mock('../giftCardSection', () => 'GiftCardSection');
+jest.mock('../GiftOptions', () => 'GiftOptions');
```
Comment (supernova-at) [inline packages/venia-ui/lib/components/CartPage/PriceAdjustments/__tests__/priceAdjustments.spec.js]: Now that the children get rendered regardless of whether their section is open or not, we have to mock them all.

**Comment by supernova-at** [comment]
PR Updated:
  * Updates `priceAdjustments` unit tests and snapshots

**Comment by dpatil-magento** [comment]
@supernova-at Not sure how its related but as demo'ed happens only with this PR, works on `develop`

When try to sign-out from cart page, sign-out does not work and console has this error.

`vendors.e2bb76dc931329fff8e3.js:1059 Uncaught (in promise) Error: Network error: Cannot read property 'include_gift_receipt' of undefined
    at new ApolloError (vendors.e2bb76dc931329fff8e3.js:1059)
    at vendors.e2bb76dc931329fff8e3.js:1142
    at async Promise.all (index 6)`

**Comment by supernova-at** [comment]
Hey @dpatil-magento I was able to reproduce that issue on `develop` so I have created a separate issue for it and I think this one can be merged (since it didn't introduce the issue).

**Pull request merged**